pyseir/cli.py

```diff
@@ -56,14 +57,37 @@ def _generate_infection_rate_metric(regions: List[infer_rt.RegionalInput]) -> pd
     Apply infer_rt.run_rt_for_fips for each region in regions and return a combined results table
     with a unique row for each region+date combination.
     """
+    if not regions:
+        return pd.DataFrame()
+
     with Pool(maxtasksperchild=10) as p:
-        results = p.map(infer_rt.run_rt, regions)
+        results = pd.concat(p.map(infer_rt.run_rt, regions))
 
-    if results:
-        return pd.concat(results)
-    else:
+    if results.empty:
         return pd.DataFrame()
 
+    need_patch = set(results.fips) & set(NEW_ORLEANS_FIPS)
+    if need_patch:
+        logging.info("Applying New Orleans Patch")
+        unpatched = results.loc[~results.fips.isin(need_patch), :]
+
+        # Aggregate the results created so far into one timeseries of metrics in a DataFrame
+        nola_rt_inferences = pyseir.rt.patches.patch_aggregate_rt_results(need_patch)
```

Comment:
I'm not sure this code captures the correct logic if you only pass a subset of the NEW_ORLEANS_FIPS.

The patch should still be the entire NEW_ORLEANS_FIPS set, even if only a subset of those fips are being returned.

`pyseir.rt.patches.patch_aggregate_rt_results(NEW_ORLEANS_FIPS)`

Reply:
For sure, that was a missing comment. Added as a warning because it is not expected to happen during prod runs.